charts/opentelemetry-collector/values.yaml

```diff
@@ -312,6 +312,11 @@ serviceMonitor:
   extraLabels: {}
   #  release: kube-prometheus-stack
 
+# PodDisruptionBudget (pdb) is used only if standaloneCollector enabled
+pdb:
```

Comment:
pdb may be too ambiguous in scope of values.yaml. Let's use full name

Reply:
Good idea! I changed it to PodDisruptionBudget.